### DISCUSSION TITLE
Get JWT token

### DISCUSSION THREAD

**Discussion by jooe81** [fsharplang-ru/pulsar-client-dotnet#166] [Q&A]
Hi there,
I'm using this Pulsar Client library and I'm very thankful for that, but I have specific request on my project to get JWT token which is generated in communication with broker. So, my question is could you implement getter for that data, please?
Thanks in advance!

**Comment by Lanayx** [comment]
Hello! Are you talking about Oauth2? There is an abstract class Authentication that you can implement using your custom requirements. Your implementation can be based on the default AuthenticationOauth2 class
